legacy/src/api/assets/tokens.md

```diff
@@ -34,13 +190,6 @@ Token value may be set in multiple currencies and is the same for all tokens in
   })
 {% endreqspec %}
 
-{% h4 Fields %}
-
-|     Field      |  Type  |                                      Description                                       |
-| :------------- | :----- | :------------------------------------------------------------------------------------- |
-| collectionId   | String | The token collection that will govern the branding and redemption rules for the token. |
-| idempotencyKey | String | Client-supplied identifier that prevents double creation.                              |
```

Comment:
We can leave the fields here. These is the fields that we pass into the Create Token API, not the model of a token object :)